x/launch/genesis.go

```diff
@@ -28,18 +34,26 @@ func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *types.GenesisState {
 	genesis := types.DefaultGenesis()
 
 	// this line is used by starport scaffolding # genesis/module/export
+
+	// Get all chain
+	chainList := k.GetAllChain(ctx)
+	for _, elem := range chainList {
+		elem := elem
```

Comment:
```suggestion
```

Reply:
I responded on another PR, this line is actually necessary